plugin/https/https_test.go

```diff
@@ -0,0 +1,130 @@
+package https
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/coredns/coredns/plugin/pkg/dnstest"
+	"github.com/coredns/coredns/plugin/test"
+	"github.com/miekg/dns"
+	"github.com/stretchr/testify/require"
+)
+
+const from = "."
```

Comment:
this doesn't look like a very useful constant

Reply:
removed this constant